tenderly-suite/token-transfers/token-transfers/handleTokenTransfer.js

```diff
@@ -0,0 +1,156 @@
+
+        try {
+            const result = await axios.get(COIN_MARKET_QUOTES_URL, { params, headers });
+
+            if (result.data.data[symbol].length !== 0) {
+                tokenPrice = result.data.data[symbol][0].quote.USD.price;
+                tokenThreshold = TOKEN_THRESHOLD_WITH_PRICE;
+                totalAmount = tokenTransferAmount * tokenPrice;
+            } else {
+                tokenThreshold = TOKEN_THRESHOLD_WITHOUT_PRICE;
+                totalAmount = tokenTransferAmount;
+            }
+        } catch (error) {
+            console.log('CMC token quote requested: ', symbol);
+            console.log('CMC error status: ', error.response.status);
+            console.log('CMC error response: ', error.response);
+            throw Error('ERROR_IN_FETCHING_PRICE');
+        }
+
+        let tempSeverity = 0;
+
+        if (totalAmount > tokenThreshold[2]) {
+            tempSeverity = 3;
+        } else if (totalAmount > tokenThreshold[1]) {
+            tempSeverity = 2;
+        } else if (totalAmount > tokenThreshold[0]) {
+            tempSeverity = 1;
+        }
+
+        if (tempSeverity) {
+            if (tempSeverity > severity) {
+                severity = tempSeverity;
+            }
+
+            amountsAboveThreshold.push(totalAmount);
+            prices.push(tokenPrice);
+            symbols.push(symbol);
+        }
+    }
+
+    if (severity) {
+        try {
+            await axios.post(
+                PAGER_DUTY_ALERT_URL,
+                {
+                    routing_key: await context.secrets.get('PD_ROUTING_KEY'),
+                    event_action: 'trigger',
+                    payload: {
+                        summary: 'Token tranfer amount crossed threshold',
+                        source: `${chainName}-ITS-${its.address}`,
+                        severity: getSeverityString(severity),
+                        custom_details: {
+                            timestamp: Date.now(),
+                            chain_name: chainName,
+                            transaction: event.hash,
+                            transfer_info: {
+                                amounts: amountsAboveThreshold,
+                                prices,
+                                symbols,
+                            },
+                            payload: event,
+                        },
+                    },
+                },
+                {},
+            );
+        } catch (error) {
+            console.log('PD error status: ', error.response.status);
+            console.log('PD error response: ', error.response.data);
+            throw Error('TOKEN_TRANSFER_ALERT_FAILED');
+        }
+    }
+};
+
+function getSeverityString(severity) {
+    if (severity === 3) {
+        return 'critical';
+    }
+
+    if (severity === 2) {
+        return 'warning';
+    }
+
+    if (severity === 1) {
+        return 'info';
+    }
+
+    return '';
```

Comment:
This seems to be unreachable code, if you decide to keep this function you can probably just return 'info' if the severity is not 2 or 3. If you want to explicitly check for each number for readability maybe you should throw an error if severity is not one of those 3 values, because something must have gone wrong at that point.